doc/news/DM-48022.feature.rst

```diff
@@ -0,0 +1,6 @@
+Deprecate `ignore_m1m3` property.
```

Comment:
There's a "removal" type you can probably use to document this one. 

Please, split both of these notes in 2 different files.